<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAircraftModelValidationComponent</class>
 <widget class="QFrame" name="CAircraftModelValidationComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model validation</string>
  </property>
  <layout class="QVBoxLayout" name="vl_AircraftModelValidation" stretch="0,1">
   <item>
    <widget class="QLabel" name="lbl_Summay">
     <property name="text">
      <string>Summary will go here</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tw_CAircraftModelValidationComponent">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tb_InvalidModels">
      <attribute name="title">
       <string>Invalid models</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_InvalidModels">
       <item>
        <widget class="swift::gui::views::CAircraftModelView" name="tvp_InvalidModels">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wi_Buttons" native="true">
         <layout class="QGridLayout" name="gl_Buttons">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="swift::gui::components::CSimulatorSelector" name="comp_Simulator">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pb_RemoveInvalid">
            <property name="text">
             <string>remove invalid</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pb_TriggerValidation">
            <property name="text">
             <string>trigger validation</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QWidget" name="wi_Enable" native="true">
            <layout class="QHBoxLayout" name="hl_Enable">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="cb_EnableStartupCheck">
               <property name="text">
                <string>check at startup</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_OnlyIfErrorsOrWarnings">
               <property name="text">
                <string>only if errors/warnings</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="hs_Enable">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="pb_RemoveSelected">
            <property name="text">
             <string>remove selected</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pb_TempDisableInvalid">
            <property name="text">
             <string>temp.disable invalid</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="pb_TempDisableSelected">
            <property name="text">
             <string>temp.disable selected</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pb_Help">
            <property name="text">
             <string>help</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_Messages">
      <attribute name="title">
       <string>Messages</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_Messages">
       <item>
        <widget class="swift::gui::components::CStatusMessagesDetail" name="comp_Messages">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CStatusMessagesDetail</class>
   <extends>QFrame</extends>
   <header>gui/components/statusmessagesdetail.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::views::CAircraftModelView</class>
   <extends>QTableView</extends>
   <header>gui/views/aircraftmodelview.h</header>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tw_CAircraftModelValidationComponent</tabstop>
  <tabstop>tvp_InvalidModels</tabstop>
  <tabstop>pb_Help</tabstop>
  <tabstop>pb_TriggerValidation</tabstop>
  <tabstop>pb_TempDisableInvalid</tabstop>
  <tabstop>pb_TempDisableSelected</tabstop>
  <tabstop>cb_EnableStartupCheck</tabstop>
  <tabstop>cb_OnlyIfErrorsOrWarnings</tabstop>
  <tabstop>pb_RemoveInvalid</tabstop>
  <tabstop>pb_RemoveSelected</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
